src/components/course/post/PostComment.tsx

```diff
@@ -1,40 +1,38 @@
-import { Avatar, Box, Grid, Typography } from '@mui/material';
-import { UserModel } from 'models/UserModel';
-import React from 'react';
+import { Avatar, Box, Grid, Tooltip, Typography } from "@mui/material";
+import { UserModel } from "models/UserModel";
+import React from "react";
+import { getAvatar } from "utils/utils";
 
 const commonStyles = {
-  bgcolor: 'background.paper',
-    borderColor: 'text.primary',
-    border: 1,
-    borderRadius: '16px',
-    padding: '10px',
-    paddingBotton: "20px",
-    maxWidth: '80vw',
-    overflowWrap: 'break-word', 
-    wordWrap: 'break-word', 
+  bgcolor: "background.paper",
+  borderColor: "text.primary",
+  border: 1,
+  borderRadius: "16px",
+  padding: "10px",
+  paddingBotton: "20px",
+  maxWidth: "80vw",
+  overflowWrap: "break-word",
+  wordWrap: "break-word",
 };
 
 export interface CommentValues {
   user: UserModel;
   text: string | null;
 }
 
-export default function Comment ({
-    user,
-    text,
-  }: CommentValues) {
-  
+export default function Comment({ user, text }: CommentValues) {
   return (
-    <Grid container wrap="nowrap" spacing={2} style={{ padding: "5px"}}>
-        <Grid item>
-          <Avatar alt="Remy Sharp" src={user.avatar} />
-        </Grid>
-        <Grid justifyContent="left" item xs zeroMinWidth>
-        <Box sx={{ ...commonStyles, borderRadius: '16px' }}>
-          <Typography sx={{ marginLeft: "10px" }}>{text}
-          </Typography>
+    <Grid container wrap="nowrap" spacing={2} style={{ padding: "5px" }}>
+      <Grid item>
+        <Tooltip title={user.username}>
+          <Avatar alt="Remy Sharp" src={getAvatar(user.avatar)} />
```

Comment:
if we have the alt attribute maybe change it to "user avatar" instead of "Remy Sharp", which does not add any additonal value